frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarContainer.tsx

```diff
@@ -187,5 +188,8 @@ export default _.compose(
     query: () => ({ tree: true, "exclude-archived": true }),
     loadingAndErrorWrapper: false,
   }),
+  Databases.loadList({
+    loadingAndErrorWrapper: false,
+  }),
```

Comment:
Before, `MainNavbarContainer` was relying on some other root component to fetch databases. That didn't let me add proper test coverage for the "Browse data" link as it depends on connected databases state